earthfile2llb/interpreter.go

```diff
@@ -37,6 +37,7 @@ type Interpreter struct {
 	isWith          bool
 	pushOnlyAllowed bool
 	local           bool
+	denyParallel    bool
```

Comment:
The default value of a bool is false, so this should default to allowing parallelism, unless explicitly prohibited, as it is for `LOCALLY` blocks (without `--allow-parallelism`.